terraso_backend/apps/soil_id/models/depth_default_interval.py

```diff
@@ -0,0 +1,22 @@
+# Copyright © 2021-2023 Technology Matters
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see https://www.gnu.org/licenses/.
+from apps.core.models import BaseModel
+from django.db import models
+from apps.project_management.models.sites import Site
+from apps.project_management.models import Project
+
+class DepthDefaultInterval(BaseModel):
+    site = models.OneToOneField(Site)
+    project = models.OneToOneField(Project)
```

Comment:
suggestion: i think this code is more so related to https://github.com/techmatters/terraso-backend/issues/622 than this issue so we can leave this file out of this change!